tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -1827,7 +1826,6 @@ declare namespace ts {
         NoTruncation = 1,
         WriteArrayAsGenericType = 2,
         WriteDefaultSymbolWithoutName = 4,
-        UseStructuralFallback = 8,
```

Comment:
@weswigham, this was changed in #21203. Is this intended to be part of the public API?